docs/src/docs/usage/faq.mdx

```diff
@@ -18,17 +18,11 @@ Run `golangci-lint` in CI and check the exit code. If it's non-zero - fail the b
 
 See [how to properly install `golangci-lint` in CI](/usage/install#ci-installation)
 
-## Which go versions are supported
+## Which Go versions are supported
 
-Short answer: Go 1.12 and newer are officially supported.
+The same as the Go team (the 2 last minor versions)
 
-Long answer:
-
-1. go < 1.9 isn't supported
-2. go1.9 is officially supported by golangci-lint <= v1.10.2
-3. go1.10 is officially supported by golangci-lint <= 1.15.0.
-4. go1.11 is officially supported by golangci-lint <= 1.17.1.
-5. go1.12+ are officially supported by the latest version of golangci-lint (>= 1.18.0).
+go1.18+ is officially supported since golangci-lint v1.45.0.
```

Comment:
I think it's worth mentioning that even though Go 1.18 is supported from `v1.45.0` not every linter in `golangci-lint` will work. Just like we did with the error message I think it's a good idea to link to https://github.com/golangci/golangci-lint/issues/2649 as long as the issue is still open.